apps/andi/test/integration/andi_web/live/extract_http_step_form_test.exs

```diff
@@ -199,6 +199,7 @@ defmodule AndiWeb.ExtractHttpStepFormTest do
       extract_step_form_view = find_live_child(view, "extract_step_form_editor")
       test_url_button = element(extract_step_form_view, "#step-#{extract_step_id} button", "Test")
       render_click(test_url_button)
+      Process.sleep(3000)
```

Comment:
Can we add that #todo: Refactor events.... comment to this sleep as well? (Only if other changes are needed as well)